ajax/libs/simplelightbox/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "simplelightbox",
+  "filename": "simple-lightbox.min.js",
+  "version": "1.7.2",
+  "description": "Touch-friendly image lightbox for mobile and desktop with jQuery",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/andreknieriem/simplelightbox.git"
+  },
+  "keywords": [
+    "lightbox",
+    "modal",
+    "gallery",
+    "jquery-plugin",
+    "touchfriendly",
+    "responsive",
+    "popup",
+    "dialog"
+  ],
+  "author": {
+    "name": "Andre Rinas",
+    "url": "http://www.andreknieriem.de",
+    "email": "info@andreknieriem.de"
+  },
+  "license": "MIT",
+  "npmName": "simplelightbox",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.+(js|css)"
```

Comment:
I would like to exclude `scss` instead of adding `js|css`, or maybe you use `js|css|map` instead will be better.